assets/js/hooks/useMonitorInternetConnection.js

```diff
@@ -63,7 +63,12 @@ export function useMonitorInternetConnection() {
 
```

Comment:
We can remove all of the response checking above because we're only concerned with it throwing or not.